build.gradle

```diff
@@ -23,7 +28,17 @@ targetCompatibility = '11'
 intellij {
     version = ideaVersion //for a full list of IntelliJ IDEA releases please see https://www.jetbrains.com/intellij-repository/releases
     pluginName = 'org.jboss.tools.intellij.openshift'
-    plugins = ['java', 'terminal', 'JavaScriptDebugger', 'JavaScript', 'com.intellij.css', 'yaml', 'com.redhat.devtools.intellij.telemetry:0.0.3.33', 'com.redhat.devtools.intellij.kubernetes:0.6.1.167']
+    // use '<git repo clone>/build/idea-sandbox/plugins/<plugin>' if working from source
+    plugins = [
+            'java',
+            'terminal',
+            'JavaScriptDebugger',
+            'JavaScript',
+            'com.intellij.css',
+            'yaml',
+            'com.redhat.devtools.intellij.telemetry:0.0.3.33',
+            '/Users/andredietisheim/Documents/jboss-workspaces/intellij-kubernetes/build/idea-sandbox/plugins/Kubernetes by Red Hat'
```

Comment:
@sbouchet: to build it locally please replace '/Users/andredietisheim/Documents/jboss-workspaces/intellij-kubernetes/build/idea-sandbox/plugins/Kubernetes by Red Hat' by the path to your locally built artifacts.
Once `intellij-kubernetes` with `kubernetes-client` `6.4.0` this can be replaced by the marketplace version.